compiler/dyno/include/chpl/queries/query-impl.h

```diff
@@ -323,11 +334,28 @@ Context::queryEnd(
     this->updateResultForQueryMapR(queryMap, r, tupleOfArgs,
                                    std::move(result), /* forSetter */ false);
 
-  if (enableDebugTrace) {
-    bool changed = ret->lastChanged == this->currentRevisionNumber;
-    printf("QUERY END       %s (", traceQueryName);
-    queryArgsPrint(tupleOfArgs);
-    printf(") %s %p\n", changed?"UPDATED":"NO CHANGE", r);
+  if (enableDebugTrace
+      && std::find(queryTraceIgnoreQueries.begin(),
+                   queryTraceIgnoreQueries.end(),
+                   traceQueryName) == queryTraceIgnoreQueries.end()) {
+    // QUERY END
+    setQueryDepthColor(queryTraceDepth, std::cout);
+    std::cout << queryTraceDepth
+              << clearTerminalColor()
+              << "   " << traceQueryName
+              << " ";
+    if (ret->lastChanged == this->currentRevisionNumber) {
+      setTerminalColor(yellow, std::cout);
+      std::cout << "UPDATED";
+    } else {
+      setTerminalColor(green, std::cout);
+      std::cout << "NO CHANGE";
```

Comment:
I think we should probably have the the unchanged path be `unchanged` or something that doesn't scream, so that way the `UPDATED` queries stand out the most. Maybe red would work to scream even louder? Or is red reserved for something else? (CATASTROPHIC FAILURE 😄)